<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="4" total="17" passed="9" failed="1" skipped="3">
  <reporter-output>
  </reporter-output>
  <suite started-at="2025-08-28T15:26:59Z" name="Suite" finished-at="2025-08-28T15:29:09Z" duration-ms="130538">
    <groups>
    </groups>
    <test started-at="2025-08-28T15:26:59Z" name="Test" finished-at="2025-08-28T15:29:09Z" duration-ms="130538">
      <class name="tests.LoginTest">
        <test-method is-config="true" signature="beforeClass(org.testng.ITestContext)[pri:0, instance:tests.LoginTest@25bcd0c7]" started-at="2025-08-28T15:26:59Z" name="beforeClass" finished-at="2025-08-28T15:27:02Z" duration-ms="2806" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[org.testng.TestRunner@4e96cb04]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeClass -->
        <test-method signature="LoginPageTest1()[pri:0, instance:tests.LoginTest@25bcd0c7]" started-at="2025-08-28T15:27:02Z" name="LoginPageTest1" finished-at="2025-08-28T15:27:06Z" duration-ms="4536" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- LoginPageTest1 -->
        <test-method is-config="true" signature="afterClass()[pri:0, instance:tests.LoginTest@25bcd0c7]" started-at="2025-08-28T15:27:06Z" name="afterClass" finished-at="2025-08-28T15:27:07Z" duration-ms="261" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterClass -->
        <test-method is-config="true" signature="beforeClass(org.testng.ITestContext)[pri:0, instance:tests.LoginTest@25bcd0c7]" started-at="2025-08-28T15:27:07Z" name="beforeClass" finished-at="2025-08-28T15:27:10Z" duration-ms="3360" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[org.testng.TestRunner@4e96cb04]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeClass -->
        <test-method signature="LoginPageTest2()[pri:0, instance:tests.LoginTest@25bcd0c7]" started-at="2025-08-28T15:27:10Z" name="LoginPageTest2" finished-at="2025-08-28T15:27:15Z" duration-ms="4476" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- LoginPageTest2 -->
        <test-method is-config="true" signature="afterClass()[pri:0, instance:tests.LoginTest@25bcd0c7]" started-at="2025-08-28T15:27:15Z" name="afterClass" finished-at="2025-08-28T15:27:15Z" duration-ms="228" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterClass -->
        <test-method is-config="true" signature="beforeClass(org.testng.ITestContext)[pri:0, instance:tests.LoginTest@25bcd0c7]" started-at="2025-08-28T15:27:15Z" name="beforeClass" finished-at="2025-08-28T15:27:17Z" duration-ms="2136" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[org.testng.TestRunner@4e96cb04]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeClass -->
        <test-method signature="LoginPageTest3()[pri:0, instance:tests.LoginTest@25bcd0c7]" started-at="2025-08-28T15:27:17Z" name="LoginPageTest3" finished-at="2025-08-28T15:27:21Z" duration-ms="4470" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- LoginPageTest3 -->
        <test-method is-config="true" signature="afterClass()[pri:0, instance:tests.LoginTest@25bcd0c7]" started-at="2025-08-28T15:27:21Z" name="afterClass" finished-at="2025-08-28T15:27:22Z" duration-ms="233" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterClass -->
        <test-method is-config="true" signature="beforeClass(org.testng.ITestContext)[pri:0, instance:tests.LoginTest@25bcd0c7]" started-at="2025-08-28T15:27:22Z" name="beforeClass" finished-at="2025-08-28T15:27:24Z" duration-ms="2254" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[org.testng.TestRunner@4e96cb04]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeClass -->
        <test-method signature="LoginPageTest4()[pri:0, instance:tests.LoginTest@25bcd0c7]" started-at="2025-08-28T15:27:24Z" name="LoginPageTest4" finished-at="2025-08-28T15:27:28Z" duration-ms="4414" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- LoginPageTest4 -->
        <test-method is-config="true" signature="afterClass()[pri:0, instance:tests.LoginTest@25bcd0c7]" started-at="2025-08-28T15:27:28Z" name="afterClass" finished-at="2025-08-28T15:27:29Z" duration-ms="265" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterClass -->
        <test-method is-config="true" signature="beforeClass(org.testng.ITestContext)[pri:0, instance:tests.LoginTest@25bcd0c7]" started-at="2025-08-28T15:27:29Z" name="beforeClass" finished-at="2025-08-28T15:27:31Z" duration-ms="2207" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[org.testng.TestRunner@4e96cb04]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeClass -->
        <test-method signature="LoginPageTest5()[pri:0, instance:tests.LoginTest@25bcd0c7]" started-at="2025-08-28T15:27:31Z" name="LoginPageTest5" finished-at="2025-08-28T15:27:35Z" duration-ms="4591" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- LoginPageTest5 -->
        <test-method is-config="true" signature="afterClass()[pri:0, instance:tests.LoginTest@25bcd0c7]" started-at="2025-08-28T15:27:35Z" name="afterClass" finished-at="2025-08-28T15:27:36Z" duration-ms="225" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterClass -->
      </class> <!-- tests.LoginTest -->
      <class name="tests.CartTest">
        <test-method is-config="true" signature="beforeClass(org.testng.ITestContext)[pri:0, instance:tests.CartTest@32cb636e]" started-at="2025-08-28T15:27:36Z" name="beforeClass" finished-at="2025-08-28T15:27:38Z" duration-ms="2526" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[org.testng.TestRunner@4e96cb04]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeClass -->
        <test-method signature="addItemToCartTest()[pri:0, instance:tests.CartTest@32cb636e]" started-at="2025-08-28T15:27:38Z" name="addItemToCartTest" finished-at="2025-08-28T15:27:43Z" duration-ms="4952" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- addItemToCartTest -->
        <test-method is-config="true" signature="afterClass()[pri:0, instance:tests.CartTest@32cb636e]" started-at="2025-08-28T15:27:43Z" name="afterClass" finished-at="2025-08-28T15:27:43Z" duration-ms="238" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterClass -->
        <test-method is-config="true" signature="beforeClass(org.testng.ITestContext)[pri:0, instance:tests.CartTest@32cb636e]" started-at="2025-08-28T15:27:43Z" name="beforeClass" finished-at="2025-08-28T15:27:45Z" duration-ms="1611" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[org.testng.TestRunner@4e96cb04]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeClass -->
        <test-method signature="addMultipleItemsToCart()[pri:0, instance:tests.CartTest@32cb636e]" started-at="2025-08-28T15:27:45Z" name="addMultipleItemsToCart" finished-at="2025-08-28T15:28:12Z" duration-ms="26912" status="SKIP">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [2] but found [null]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [2] but found [null]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:453)
at org.testng.Assert.assertEquals(Assert.java:463)
at tests.CartTest.addMultipleItemsToCart(CartTest.java:39)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- addMultipleItemsToCart -->
        <test-method is-config="true" signature="afterClass()[pri:0, instance:tests.CartTest@32cb636e]" started-at="2025-08-28T15:28:12Z" name="afterClass" finished-at="2025-08-28T15:28:12Z" duration-ms="28" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterClass -->
        <test-method is-config="true" signature="beforeClass(org.testng.ITestContext)[pri:0, instance:tests.CartTest@32cb636e]" started-at="2025-08-28T15:28:12Z" name="beforeClass" finished-at="2025-08-28T15:28:15Z" duration-ms="3494" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[org.testng.TestRunner@4e96cb04]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeClass -->
        <test-method signature="addMultipleItemsToCart()[pri:0, instance:tests.CartTest@32cb636e]" started-at="2025-08-28T15:28:15Z" name="addMultipleItemsToCart" finished-at="2025-08-28T15:28:24Z" duration-ms="8154" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- addMultipleItemsToCart -->
        <test-method is-config="true" signature="afterClass()[pri:0, instance:tests.CartTest@32cb636e]" started-at="2025-08-28T15:28:24Z" name="afterClass" finished-at="2025-08-28T15:28:24Z" duration-ms="206" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterClass -->
        <test-method is-config="true" signature="beforeClass(org.testng.ITestContext)[pri:0, instance:tests.CartTest@32cb636e]" started-at="2025-08-28T15:28:24Z" name="beforeClass" finished-at="2025-08-28T15:28:25Z" duration-ms="1518" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[org.testng.TestRunner@4e96cb04]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeClass -->
        <test-method signature="addMultipleItemsToCartAndRemove()[pri:0, instance:tests.CartTest@32cb636e]" started-at="2025-08-28T15:28:25Z" name="addMultipleItemsToCartAndRemove" finished-at="2025-08-28T15:28:30Z" duration-ms="5172" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- addMultipleItemsToCartAndRemove -->
        <test-method is-config="true" signature="afterClass()[pri:0, instance:tests.CartTest@32cb636e]" started-at="2025-08-28T15:28:30Z" name="afterClass" finished-at="2025-08-28T15:28:31Z" duration-ms="278" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterClass -->
        <test-method is-config="true" signature="beforeClass(org.testng.ITestContext)[pri:0, instance:tests.CartTest@32cb636e]" started-at="2025-08-28T15:28:31Z" name="beforeClass" finished-at="2025-08-28T15:28:34Z" duration-ms="3100" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[org.testng.TestRunner@4e96cb04]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeClass -->
        <test-method signature="verifyContentsOnCartandInventory()[pri:0, instance:tests.CartTest@32cb636e]" started-at="2025-08-28T15:28:34Z" name="verifyContentsOnCartandInventory" finished-at="2025-08-28T15:28:42Z" duration-ms="8151" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyContentsOnCartandInventory -->
        <test-method is-config="true" signature="afterClass()[pri:0, instance:tests.CartTest@32cb636e]" started-at="2025-08-28T15:28:42Z" name="afterClass" finished-at="2025-08-28T15:28:42Z" duration-ms="279" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterClass -->
      </class> <!-- tests.CartTest -->
      <class name="tests.CheckoutTest">
        <test-method is-config="true" signature="beforeClass(org.testng.ITestContext)[pri:0, instance:tests.CheckoutTest@63cd604c]" started-at="2025-08-28T15:28:42Z" name="beforeClass" finished-at="2025-08-28T15:28:44Z" duration-ms="1914" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[org.testng.TestRunner@4e96cb04]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeClass -->
        <test-method signature="testcase5()[pri:0, instance:tests.CheckoutTest@63cd604c]" started-at="2025-08-28T15:28:44Z" name="testcase5" finished-at="2025-08-28T15:28:51Z" duration-ms="6946" status="SKIP">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [58.29] but found [158.29]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [58.29] but found [158.29]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:442)
at tests.CheckoutTest.testcase5(CheckoutTest.java:123)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testcase5 -->
        <test-method is-config="true" signature="afterClass()[pri:0, instance:tests.CheckoutTest@63cd604c]" started-at="2025-08-28T15:28:51Z" name="afterClass" finished-at="2025-08-28T15:28:51Z" duration-ms="249" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterClass -->
        <test-method is-config="true" signature="beforeClass(org.testng.ITestContext)[pri:0, instance:tests.CheckoutTest@63cd604c]" started-at="2025-08-28T15:28:51Z" name="beforeClass" finished-at="2025-08-28T15:28:54Z" duration-ms="2175" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[org.testng.TestRunner@4e96cb04]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeClass -->
        <test-method signature="testcase5()[pri:0, instance:tests.CheckoutTest@63cd604c]" started-at="2025-08-28T15:28:54Z" name="testcase5" finished-at="2025-08-28T15:29:01Z" duration-ms="7052" status="SKIP">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [58.29] but found [158.29]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [58.29] but found [158.29]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:442)
at tests.CheckoutTest.testcase5(CheckoutTest.java:123)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.retryFailed(Invoker.java:839)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1010)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testcase5 -->
        <test-method is-config="true" signature="afterClass()[pri:0, instance:tests.CheckoutTest@63cd604c]" started-at="2025-08-28T15:29:01Z" name="afterClass" finished-at="2025-08-28T15:29:01Z" duration-ms="269" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterClass -->
        <test-method is-config="true" signature="beforeClass(org.testng.ITestContext)[pri:0, instance:tests.CheckoutTest@63cd604c]" started-at="2025-08-28T15:29:01Z" name="beforeClass" finished-at="2025-08-28T15:29:03Z" duration-ms="1757" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[org.testng.TestRunner@4e96cb04]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeClass -->
        <test-method signature="testcase5()[pri:0, instance:tests.CheckoutTest@63cd604c]" started-at="2025-08-28T15:29:03Z" name="testcase5" finished-at="2025-08-28T15:29:09Z" duration-ms="6396" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [58.29] but found [158.29]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [58.29] but found [158.29]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:442)
at tests.CheckoutTest.testcase5(CheckoutTest.java:123)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.retryFailed(Invoker.java:839)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1010)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testcase5 -->
        <test-method is-config="true" signature="afterClass()[pri:0, instance:tests.CheckoutTest@63cd604c]" started-at="2025-08-28T15:29:09Z" name="afterClass" finished-at="2025-08-28T15:29:09Z" duration-ms="204" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterClass -->
      </class> <!-- tests.CheckoutTest -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
